api/src/pcapi/sandboxes/scripts/creators/test_cases/__init__.py

```diff
@@ -85,3 +87,66 @@ def create_offers_with_gtl_id(gtl_id: str, size_per_gtl: int, venue: offerers_mo
             size=random.randint(1, 10),
             offer=offer,
         )
+
+
+def create_offers_with_same_ean() -> None:
+    offers = []
+    ean = Fake.ean13()
+    author = Fake.name()
+    product = offers_factories.ProductFactory(
+        name="Le livre du pass Culture",
+        subcategoryId=subcategories_v2.LIVRE_PAPIER.id,
+        extraData={
+            "ean": ean,
+            "author": author,
+        },
+    )
+    for venue_data in venues:
+        offers.append(
+            offers_factories.OfferFactory(
+                product=product,
+                isActive=True,
+                subcategoryId=subcategories_v2.LIVRE_PAPIER.id,
+                venue=offerers_factories.VenueFactory(
+                    name=venue_data["name"],
+                    venueTypeCode=offerers_models.VenueTypeCode.BOOKSTORE,
+                    isPermanent=True,
+                    latitude=venue_data["latitude"],
+                    longitude=venue_data["longitude"],
+                    address=venue_data["address"],
+                    postalCode=venue_data["postalCode"],
+                    city=venue_data["city"],
+                    departementCode=venue_data["departementCode"],
+                ),
+                extraData={
+                    "ean": ean,
+                    "author": author,
+                },
+            )
+        )
+        for offer in offers:
+            offers_factories.StockFactory.create_batch(
+                size=random.randint(1, 10),
+                offer=offer,
+            )
+    for _ in range(10):
+        ean = Fake.ean13()
+        author = Fake.name()
+        create_offer_with_ean(ean, random.choice(offers).venue, author=author)
+
+
+def create_offer_with_ean(ean: str, venue: offerers_models.Venue, author: str) -> None:
+    product = offers_factories.ProductFactory(
+        subcategoryId=subcategories_v2.LIVRE_PAPIER.id,
+        extraData={"ean": ean, "author": author},
+    )
+    offer = offers_factories.OfferFactory(
+        isActive=True,
+        product=product,
+        extraData={"ean": ean, "author": author},
+        venue=venue,
+    )
+    offers_factories.StockFactory.create_batch(
+        size=random.randint(1, 10),
```

Comment:
Notre modèle le permet mais pas le fonctionnement de l'app. Une offre produit (not isEvent) ne doit avoir qu'un seul stock actif en même temps.
Il existe déjà des produits avec EAN et des offres liés à ces produits, elles ne vous conviennent pas ? Vous en voulez plus ou sur des lieux spécifiques ?